state-chain/cf-integration-tests/src/network.rs

```diff
@@ -36,6 +36,7 @@ type CfeEvent = cfe_events::CfeEvent<<Runtime as Chainflip>::ValidatorId>;
 
 // TODO: Can we use the actual events here?
 // Events from ethereum contract
+#[allow(dead_code)]
```

Comment:
This makes it seem like the whole event is unused, we can instead do:
```rust
Funded {
		node_id: NodeId,
		amount: FlipBalance,
		#[allow(dead_code)]
		total: FlipBalance,
		epoch: EpochIndex,
	},
```
or better... remove the field